sdk/containerregistry/Azure.ResourceManager.ContainerRegistry/src/Generated/Models/UnknownContainerRegistryTaskStepProperties.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System.Collections.Generic;
+
+namespace Azure.ResourceManager.ContainerRegistry.Models
+{
+    /// <summary> The UnknownContainerRegistryTaskStepProperties. </summary>
+    internal partial class UnknownContainerRegistryTaskStepProperties : ContainerRegistryTaskStepProperties
```

Comment:
Can we rename it to `UnknownContainerRegistryTaskStep` to align with previous naming pattern where only the base class keep the `Properties` suffix?

Reply:
Currently we can't rename this model as it's auto generated and I'd not like to use customization to do that. Considering it's an internal model, I think we can merge it first and I documented this issue [here](https://github.com/Azure/autorest.csharp/issues/2678#issuecomment-1245045891) to facilitate future support for this renaming in generator.

Reply:
I see. I didn't notice it's internal. This is added by us working as a fallback when type is not set, not swaggers?

Reply:
Yes, correct.